scripts/ttfa-test.sh

```
@@ -0,0 +1,180 @@
+# Configuration
+SIMULATOR="${SIMULATOR:-iPhone 16 Pro}"
+TTFA_SUITE="${TTFA_SUITE:-quick}"
+TTFA_FORMAT="${TTFA_FORMAT:-text}"
+SKIP_BUILD="${SKIP_BUILD:-false}"
+
+# Colors
+GREEN='\033[92m'
+RED='\033[91m'
+YELLOW='\033[93m'
+CYAN='\033[96m'
+BOLD='\033[1m'
```

Shellcheck (0.11.0)

[warning] BOLD appears unused. Verify use (or export if used externally).

(SC2034)